Resources/Prototypes/Nyanotrasen/Entities/Objects/Devices/CircuitBoards/production.yml

```diff
@@ -33,16 +33,10 @@
         MatterBin: 1
         Manipulator: 1
       materialRequirements:
-        Glass: 1
         Cable: 1
-        Diamond: 10
-      tagRequirements:
-        BorgArm:
-          Amount: 3
-          DefaultPrototype: LeftArmBorg
-          ExamineName: borg arm
```

Comment:
So first things first, why did you delete all the requirements to build the RE besides leaving a singular cable as a requirement.